kolibri/core/assets/src/views/AttemptLogList.vue

```diff
@@ -70,18 +71,24 @@
             />
             <p class="item">
               {{
-                coreString(
-                  'questionNumberLabel',
-                  {questionNumber: attemptLog.questionNumber}
-                )
+                windowIsLarge ?
+                  coreString(
+                    'questionNumberLabel',
+                    {questionNumber: attemptLog.questionNumber}
+                  )
+                  :
+                  // Add non-breaking space to preserve vertical centering
+                  "&nbsp;"
               }}
             </p>
+            <CoachContentLabel
+              v-if="windowIsLarge"
+              class="coach-content-label"
+              :value="attemptLog.num_coach_contents || 0"
+              :isTopic="false"
+              style="margin-top:-4px"
```

Comment:
You could add this rule to the `.coach-content-label` class below

Reply:
Thanks JB - changed.